spec/Section 2 -- Language.md

```diff
@@ -815,27 +816,73 @@ StringCharacter ::
   - `\u` EscapedUnicode
   - `\` EscapedCharacter
 
-EscapedUnicode :: /[0-9A-Fa-f]{4}/
+EscapedUnicode ::
+  - `{` HexDigit+ `}`
+  - HexDigit HexDigit HexDigit HexDigit
+
+HexDigit :: one of
+  - `0` `1` `2` `3` `4` `5` `6` `7` `8` `9`
+  - `A` `B` `C` `D` `E` `F`
+  - `a` `b` `c` `d` `e` `f`
 
 EscapedCharacter :: one of `"` `\` `/` `b` `f` `n` `r` `t`
 
 BlockStringCharacter ::
   - SourceCharacter but not `"""` or `\"""`
   - `\"""`
 
-Strings are sequences of characters wrapped in quotation marks (U+0022).
-(ex. {`"Hello World"`}). White space and other otherwise-ignored characters are
-significant within a string value.
+{StringValue} is a sequence of characters wrapped in quotation marks (U+0022).
```

Comment:
It is getting a bit meta but I don't think character is completely correct here. It is either a sequence of Unicode Scalar Values or a sequence of Code Points depending on how you look at it.